hkmc2Benchmarks/src/test/logs/.gitignore

```diff
@@ -0,0 +1 @@
+*.json
```

Comment:
Why such a blanket rule? Please ignore specific files instead.

Reply:
The logs folder is meant for benchmark log json output. I couldn't imagine wanting to commit the benchmark logs, since every run is bound to change it. Adding specific files will require updating .gitignore for new benchmarks that write logs, which is easy to forget.